basics/installation/environments/unix-specific.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Installing an AMP stack on Unix
+menuTitle: Unix environment
+weight: 3
+---
+
+# Installing an AMP stack on Unix
+
+PrestaShop is an Open-Source web application running on an *AMP stack (Apache - MySQL - PHP).
+
+In order for PrestaShop to run locally on your MacOS computer, you need to install an *AMP stack. 
```

Comment:
You mentioned MacOS again whereas you have instruction for MacOS above

Reply:
it was a mistake